Source/Core/VideoCommon/UberShaderCommon.cpp

```diff
@@ -119,11 +120,11 @@ void WriteVertexLighting(ShaderCode& out, APIType api_type, std::string_view wor
   out.Write("  if ({} != 0u) {{\n", BitfieldExtract<&LitChannel::enablelighting>("alphareg"));
   out.Write("    if ({} != 0u) {{\n", BitfieldExtract<&LitChannel::ambsource>("alphareg"));
   out.Write("      if ((components & ({}u << chan)) != 0u) // VB_HAS_COL0\n",
-            static_cast<u32>(VB_HAS_COL0));
+            Common::ToUnderlying(VB_HAS_COL0));
```

Comment:
These (as well as the ones in UberShaderVertex.cpp) look really sketchy though, this code would clearly break if it actually printed as signed. None of the values are actually big enough for that to happen, but this is clearly intended as unsigned.

Reply:
Multiple other places use it to modify u32s.  Perhaps the underlying type for this one should be changed to that.